google/cloud/pubsub/internal/subscription_lease_management.cc

```diff
@@ -26,10 +26,11 @@ std::chrono::seconds constexpr SubscriptionLeaseManagement::kAckDeadlineSlack;
 
 void SubscriptionLeaseManagement::Start(std::shared_ptr<BatchCallback> cb) {
   auto weak = std::weak_ptr<SubscriptionLeaseManagement>(shared_from_this());
-  child_->Start(std::make_shared<BatchCallbackWrapper>(
+  callback_ = std::make_shared<BatchCallbackWrapper>(
```

Comment:
Is `callback_` protected by `mu_`?  If not, as it seems, then I would move its declaration in the header up a couple of lines, before `mu_`.

Reply:
Nope. Done.